pkgs/top-level/all-packages.nix

```diff
@@ -3031,6 +3031,8 @@ in
       protobuf = pkgs.protobuf3_8.overrideDerivation (oldAttrs: { stdenv = clangStdenv; });
     };
 
+    rime = callPackage ../tools/inputmethods/ibus-engines/ibus-rime { };
```

Comment:
I think this should be in ibus-engines a few lines above but I am not 100% sure.

Reply:
But this is indeed in ibus-engines.

Reply:
nvm.